config/application_template.properties

```diff
@@ -37,3 +37,7 @@ box_client_redirect_uri=SERVER_URL
 
 server_url=SERVER_URL
 
+## LDAP lookup settings
+use.testUser.url=false
```

Comment:
Make sure that the default for this in the Java code is false if the property isn't set at all.